README.md

```diff
@@ -2,10 +2,12 @@
 
 A Python decorator that makes it easy to understand the error rate, response time, and production usage of any function in your code. Jump straight from your IDE to live Prometheus charts for each HTTP/RPC handler, database method, or other piece of application logic.
 
-Autometrics for Python provides a decorator that can create [Prometheus](https://prometheus.io/) metrics for your functions and class methods throughout your code base, as well as a function that will write corresponding Prometheus queries for you in a Markdown file.
+Autometrics for Python provides
 
-[See Why Autometrics?](https://github.com/autometrics-dev#why-autometrics) for more details on the ideas behind autometrics
+1. A decorator that can create [Prometheus](https://prometheus.io/) metrics for your functions and class methods throughout your code base.
+2. A helper function that will write corresponding Prometheus queries for you in a Markdown file.
```

Comment:
i'm actually not sure if the "Markdown file" bit is accurate? if so, i should add an example